spec/dummy/app/views/home/headers.rb

```diff
@@ -138,13 +135,28 @@ def main
 
     hr
 
-    docs 'Basic Page Header', %{
+    h3 'Basic Page Header'
+
+    docs 'Multi-Step Flows', %{
+      div(class: 'page_header') {
+        div(class: 'page_header_context'){
+          span '2015 Civic Innovation Conference', href: '#'
```

Comment:
You're right. But we need to specify that in the style guide. Also, a modal is probably nicer than a JS alert for this.

Overlooked the `href` attribute, thanks. Fixed